addons/website/views/website_views.xml

```diff
@@ -283,6 +283,14 @@
                             type="object" attrs="{'invisible': [('page_ids', '=', [])]}"/>
                     </header>
                 </sheet>
+                <xpath expr='//group[@groups="base.group_no_one"]' position="after">
```

Comment:
Maybe `position="inside"` instead? This seems quite a technical information, and I'm not sure a non-technical user might know what to do with such a warning.

Reply:
+1 
Message is a warning and not an error. so probably not a "danger" and should be only displayed in debug mode.
"Erratic" is too much imo.

Reply:
> if you add copy=False, need to check in each copy case where we want to keep this one (e.g. COW method).

Yep, that's why I was suggesting it for master, here we are in stable.

> Why not just add a sql contraint on (key,website_id) instead to add warning to say that user make a unexpected config.

It won't apply on databases that are already broken... but I can add it for new dbs, good point

> Do you see only one case where you want to keep 2 same key for a same website. (Except for upcoming versioning maybe?). But tomorrow is another day...

Nope, if you, the expert of website app, tells me that it makes never sense to do it, let's go for a constraint :)

Reply:
customize a report is technical, rename account_report into account_report2 and it will be broken because you don't have account_report2_document.
btw ir_ui_view form view is already in technical ;)